src/containers/App/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Route, HashRouter } from 'react-router-dom'
```

Comment:
I don't know whether we need to use browser or hash router. I've not yet investigated the difference. I can't remember why we chose hash in the first place to be honest, but I can change it and add the Switch and we can always change it back if needed

Reply:
There are two types of Router components that you can use in your React web application. The BrowserRouter and HashRouter. The former gives you a URL without the #, while the latter gives you a URL with the #.

Note: If you are building a web application that supports legacy browsers, it's recommended that you use the HashRouter.

**HashRouter** basically it uses the hash in the URL to render the component. Since I was building a static one-page website, I needed to use this.

**BrowserRouter**, it uses HTML5 history API to render the component. The history can be modified via pushState and replaceState. 

I would go Personally  for {BrowserRouter} as I do not think we would worry about older browsers and the ability to use HTML5 history API good to have

Reply:
Agreed. Also done and on the way